src/dsverifier.cpp

```diff
@@ -1087,7 +1107,7 @@ std::string prepare_bmc_command_line_ss()
   }
 
   if(dsv_strings.desired_property.size() > 0)
-    command_line += " -DPROPERTY=" + dsv_strings.desired_property;
+    command_line += " -DPROPERTY=" + dsv_strings.desired_property + " --unwind 1";
```

Comment:
You should not hard-code this `unwind` parameter to `1`. Please use the `X_SIZE` parameter.